### ISSUE TITLE
Operator '+' not supported for types 'List[Tuple[Literal[1], Literal[0]]]' and 'List[Tuple[Literal[0], Literal[0]]]

### ISSUE THREAD

**Issue by idanpa** [microsoft/pyright#441]
The following code:
``` python
p = [(1,0)]*3 + [(0,0)]*3
```

Gives this warning:

> Operator '+' not supported for types 'List[Tuple[Literal[1], Literal[0]]]' and 'List[Tuple[Literal[0], Literal[0]]]

I'm using  pyright 1.1.11 on vscode

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is now fixed in version 1.1.12, which I just published.
